cmd/sealer/cmd/utils/cluster_test.go

```diff
@@ -0,0 +1,203 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"fmt"
+	"net"
+	"testing"
+
+	"github.com/sealerio/sealer/cmd/sealer/cmd/types"
```

Comment:
usually, there should be three groups separated by blank for go-imports

the golang official package
the package for the current project
the others